src/com/projectkorra/projectkorra/airbending/Suffocate.java

```diff
@@ -220,10 +225,11 @@ public void run() {
 			}
 		}
 
-		this.animate();
+		for (final LivingEntity target : this.targets)
+			this.animate(target);
+
 		if (!this.player.isSneaking()) {
 			this.remove();
-			return;

```

Comment:
Why remove the return?

Reply:
it's unnecessary at the end of a void code block. wouldn't really impact anything if I added it back, it was just something my IDE picked up on. but I suppose if I changed every unnecessary thing about ProjectKorra that my IDE picked up on, ProjectKorra would look very different.